dolphin/common/constants.py

```diff
@@ -54,7 +54,7 @@ class StorageType(object):
 class ResourceType(IntEnum):
     STORAGE_DEVICE = 0
     STORAGE_POOL = 1
-    VOLUME = 2
+    STORAGE_VOLUME = 2
```

Comment:
Will this have corresponding impact at other places in the code?

Reply:
Related code has been modified.